pkg/controllers/externalsecret/externalsecret_controller.go

```diff
@@ -304,19 +315,18 @@ func hashMeta(m metav1.ObjectMeta) string {
 		annotations map[string]string
 		labels      map[string]string
 	}
-	h := md5.New() //nolint
-	_, _ = h.Write([]byte(fmt.Sprintf("%v", meta{
+	return utils.ObjectHash(meta{
```

Comment:
since I am using a similar hashing function elsewhere, it made sense to unify these. Note, I am using `%+v` instead of `%v` because with the `+` it will print field names as well, less potential for collision